spec/models/manageiq/providers/vmware/infra_manager/inventory/collector_spec.rb

```diff
@@ -66,18 +66,19 @@ def assert_table_counts(ems)
 
         _obj, props = collector.send(:process_object_update, object_update)
 
-        expect(props).to have_attributes(
+        expect(props).to include(
           "name"        => "Datacenters",
           "parent"      => nil,
-          "childEntity" => [datacenter]
         )
+
+        expect(props["childEntity"].first._ref).to eq(datacenter._ref)
```

Comment:
Surprised the above line needs to change at all but you might also try this if you want to keep it inline:

```ruby
"childEntity" => an_array_including(datacenter)
```

Reply:
lol I was hitting the submit button on this as @Ladas merged. Anyway @agrare you can try that if you want to make it a little cleaner but whatevers.

Reply:
Yeah this one caused a weird error where the test failed but the diff was empty

Reply:
https://github.com/ManageIQ/manageiq-providers-vmware/pull/203 fixes this